udata_front/theme/gouvfr/assets/js/types.ts

```diff
@@ -180,9 +180,16 @@ export type RefusedMembershipRequest = MembershipRequest & {
 
 export type MemberRole = "admin" | "editor";
 
+// In org endpoint we get these two private information if we have edit rights on the org.
+export type MemberUser = User & {
+  email: string;
```

Comment:
If I try on demo, it should be `string | null`. Can you confirm it ?

Reply:
I'm using [this example from demo](https://demo.data.gouv.fr/api/1/organizations/534fff81a3a7292c64a77e5c/) and it's not the case for me

Reply:
You need to have edit access to the org?

Reply:
I see the emails on this link https://demo.data.gouv.fr/api/1/organizations/534fff81a3a7292c64a77e5c/

Reply:
> You need to have edit access to the org?

It seems yes. Are you superadmin on demo ?

Reply:
The email is inside the `user` and not inside the `member`

Reply:
The field is there but it's null

Reply:
Fixed in 79645092